File: mifosng-android/src/main/java/com/mifos/objects/organisation/LoanPurpose.java
/*
 * This project is licensed under the open source MPL V2.
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.objects.organisation;

/**
 * Created by nellyk on 2/21/2016.
 */
public class LoanPurpose {

    private Integer id;
    private String name;
    private String position;


    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPosition() {
        return position;
    }

    public void setPosition(String position) {
        this.position = position;
    }

    @Override
    public String toString() {
        return "loanpurpose{" +
                "id=" + id +
                ", name='" + name + '\'' +
                ", position='" + position + '\'' +
                '}';
    }
}

File: core/database/src/main/java/com/mifos/core/objects/organisation/LoanPurpose.kt
/*
 * Copyright 2024 Mifos Initiative
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at https://mozilla.org/MPL/2.0/.
 *
 * See https://github.com/openMF/android-client/blob/master/LICENSE.md
 */
package com.mifos.core.objects.organisation

/**
 * Created by nellyk on 2/21/2016.
 */
data class LoanPurpose(
    var id: Int? = null,

    var name: String? = null,

    var position: String? = null,
)
